src/main/kotlin/no/nav/syfo/application/api/authentication/ApiFeature.kt

```diff
@@ -106,13 +116,27 @@ fun Application.installStatusPages() {
                 }
             }
 
-            val message = cause.message ?: "Unknown error"
-            call.respond(responseStatus, message)
-
             val callId = getCallId()
```

Comment:
Er det en fare for at getCallId og/eller getConsumerId kan resultere i en IndexOutOfBounds eller en annen type exception?

Reply:
Ikke såvidt jeg vet om, ganske sikker på at det kun kan bli en streng eller evt. null.

Reply:
Endre til å sende følgende melding når http-500 sendes pga feil som ikke er eksplisitt håndtert:
`"The server reported an unexpected error and cannot complete the request."`